scripts/create-sharded-conf.py

```diff
@@ -3,18 +3,35 @@
 import argparse
 import shutil
 
-def generate_keys(base_dir, shard_count, helper_name_format):
+DNS_FORMAT = "h{}-helper-shard-{}.h{}-helper-shard.default.svc.cluster.local"
```

Comment:
can you provide a pointer to k8 configuration code that expects this format?

Reply:
I can add some notes, but it's spread between the helper and service k8s templates